core/include/ldml/tsconfig.build.json

```diff
@@ -3,7 +3,7 @@
     "compilerOptions": {
         "composite": true,
         "declaration": true,
-        "module": "CommonJS",
+        "module": "es2020",
```

Comment:
Instead of doing this, we should be using `"extends": "../../../tsconfig.esm-base.json"`, and deleting `"module"` and `"moduleResolution"` entries.

Reply:
done